app/views/provider_interface/interviews/check.html.erb

```diff
@@ -1,16 +1,21 @@
-<% content_for :browser_title,  t('.page_title')  %>
+<% translation_prefix = @interview.present? ? '.update' : '' %>
```

Comment:
Consider moving this on the controller, maybe on a helper method. It's easy to miss variables that are setup in views.